contextMenu.js

```diff
@@ -135,6 +136,26 @@
                             }
                         }
 
+						//update ico
+                        if (ico) {
+                            var imgIcon = elm.find('.iw-mIcon');
+                            if (imgIcon.length) {
+								if(imgIcon[0].tagName.toLowerCase() === "span"){
+									var classList = ["iw-mIcon"];
```

Comment:
You can just write imgIcon.attr('class', 'iw-mIcon '+ico). And always accept ico as string. If multiple class is required it can be added as space seperated ('icon-add align-left')